### ISSUE TITLE
Info/Feature request

### ISSUE THREAD

**Issue by vvinci00** [concretecms-community-store/community_store#880]
Hello,

we are trying to update our site from concrete 8 to concrete 9 and we need to migrate also the community store with products and variations.

There's any known way to migrate it or could you suggest any best practice?
We are trying the migration_tool but with several issue and it not detecting community store and it's products.

thanks

**Comment by Mesuva** [comment]
As you're going from V8 to V9, you shouldn't need a migration tool for that.

It should be a case of upgrading Community Store the latest release, then upgrading from V8 to V9.
Community Store's latest release should be compatible with both v8 and v9 still.

Previously when going from V8 to V9 were some issues with upgrading attributes (in products and orders), but the last time I did this upgrade things went fine.

I'd only attempt this on a copy of a site though, to see how it all goes as a test.
What happens when you try that now?

**Comment by vvinci00** [comment]
Hi,

the main problem is that the theme we are using in V8 has been published as separate packages for V9 and not as an update. So, we have tried to update the website itself but all pages broke.

The only solution we have found is to create a new V9 instance and migrate the content

**Comment by Mesuva** [comment]
Wouldn't you be able to swap back to the default theme, remove the existing theme, do the upgrade to V9, then install the 'new' theme and apply it?

That's still going to be easier than trying to rebuilt and reconfigure a whole new site.

**Comment by vvinci00** [comment]
this what we have firstly done:
1. update  to concrete 9
2. update to php 8.3
3. install new theme 

now we have tried:
1. switch to elemental theme
2. update to concrete 9
3. update to php 8.3
4. install new theme

the result it's the same: all pages broken

**Comment by Mesuva** [comment]
What error message are you getting on the broken pages?
Can you show a full debug here?

**Comment by vvinci00** [comment]
Hi,

no error appear on php or concrete.
The problem is on the theme. 

although the theme is the same, the concrete8 (php7) theme package is different, with different handle, from the concrete9 theme package.
When we try to update, all the pages lost their settings and content's organisation and this is why we think that the only way is to create a new server and migrate the content manually.

we have also tried to use the migration tool (https://github.com/concretecms/addon_migration_tool) but we are having a lot of errors on concrete9 and section missing in the new theme package.

**Comment by Mesuva** [comment]
The migration tool isn't going to help with Community Store.

What that sounds like is that the theme is just a different theme, perhaps using the same name.

I've been updating a lot of V8 sites lately to V9, and even if a theme isn't fully updated for V9, normally it doesn't take _too_ much to work to make it compatible.  What happens when you use the existing theme in V9?

**Comment by vvinci00** [comment]
yes the name and the author it's the same (it's pixel theme) but for concrete V9 it has been published a new package instead of an update.
if we use pixel2 (officially compatibile with concrete 8) in concrete 9 we got several error on variables type in page loading.

**Comment by mlocati** [comment]
The problem is that:

1. pixel2 comes with many useful block types
2. @shahroq published a version of pixel that's compatible with concrete 9, but he changed the handle of the package

Because of 2, it's not possible to simply upgrade the pixel package.
The only solution I can see is to use the migration tool, in the hope that the handles of the block types of pixel9 are the same as the ones of pixel2.
